client/src/components/CourseNavHome/CourseNavHome.js

```diff
@@ -0,0 +1,224 @@
+import React from 'react'
```

Comment:
rename file to `Language Home`  index.js, Folder, and `ManageView` to `languagehome` adn the import in `CourseHome.js`